website/thaliawebsite/settings.py

```diff
@@ -402,6 +402,7 @@ def from_env(
     "registrations.apps.RegistrationsConfig",
     "payments.apps.PaymentsConfig",
     "singlepages.apps.SinglepagesConfig",
+    "shortlink.apps.ShortlinkConfig",
```

Comment:
```suggestion
    "shortlinks.apps.ShortlinksConfig",
```

All packages are plural.